09-soil_exploration_module.Rmd

```diff
@@ -182,142 +176,162 @@ Great! Now let's calculate some basic statistics. For example, we might want to
 
 This command follows the code structure:
 
-OBJECT %>% pull(column_name) %>% mean()
+object_name %>% pull(column_name) %>% mean()
 
-`pull()` is a command from the `tidyverse` package, so we'll need to load that library before our command.
+For this chunk of code, R will calculate the mean for the AS_EPA3051 values in the `soil.values.clean` dataset. Because we have not used the `<-` to save the mean in a new object, R will simply display the mean.
 
 ```{r, message = FALSE, warning = FALSE}
 
-library(tidyverse)
-
 soil.values.clean %>% pull(As_EPA3051) %>% mean()
 ```
 
-We can run similar commands to calculate the standard deviation (`sd`), minimum (`min`), and maximum (`max`) for the soil arsenic values.
+We can run similar commands to calculate the standard deviation (`sd`),
+minimum (`min`), and maximum (`max`) for the soil arsenic values.
 
 ```{r, message = FALSE, warning = FALSE}
 
 soil.values.clean %>% pull(As_EPA3051) %>% sd()
 soil.values.clean %>% pull(As_EPA3051) %>% min()
 soil.values.clean %>% pull(As_EPA3051) %>% max()
 ```
+The soil testing dataset contains samples from multiple geographic regions, so maybe it's more meaningful to find out what the average arsenic values are for each region. However, first we need to grab a new dataset from the `BioDIGSData` package that tells us the geographic regions of each sample.
 
-The soil testing dataset contains samples from multiple geographic regions, so maybe it's more meaningful to find out what the average arsenic values are for each region. We have to do a little bit of clever coding trickery for this using the `group_by` and `summarize` functions. First, we tell R to split our dataset up by a particular column (in this case, region) using the `group_by` function, then we tell R to summarize the mean arsenic concentration for each group. 
+Let's open `BioDIGS_metadata`, save it as `soil.meta`, and look at it.
+
+```{r, message = FALSE, warning = FALSE, echo = FALSE}
+
+soil.meta <- BioDIGS_metadata()
+```
+
+```{r, message = FALSE, warning = FALSE, eval = FALSE}
+
+soil.meta <- BioDIGS_metadata()
+
+View(soil.meta)
```

Comment:
Maybe screenshots with the `View()` commands?

Reply:
Ooo, yes!